<?xml version="1.0"?>
<launch>

	<node pkg="airo_control" type="airo_control_node" name="airo_control_node" output="screen">
        <rosparam command="load" file="$(find airo_control)/config/gazebo/fsm_gazebo.yaml" ns="fsm"/>
		<rosparam command="load" file="$(find airo_control)/config/gazebo/observer/ekf_observer_gazebo.yaml" ns="ekf_observer"/>
		<rosparam command="load" file="$(find airo_control)/config/gazebo/observer/rd3_observer_gazebo.yaml" ns="rd3_observer"/>
		<rosparam command="load" file="$(find airo_control)/config/gazebo/controller/mpc_gazebo.yaml" ns="mpc"/>
		<rosparam command="load" file="$(find airo_control)/config/gazebo/controller/backstepping_gazebo.yaml" ns="backstepping"/>
		<rosparam command="load" file="$(find airo_control)/config/gazebo/controller/slidingmode_gazebo.yaml" ns="slidingmode"/>
	</node>

</launch>